src/Renci.SshNet/PrivateKeyFile.cs

```diff
@@ -74,10 +74,27 @@ public class PrivateKeyFile : IDisposable
 
         private Key _key;
 
+        private HostAlgorithm _hostKey;
+
         /// <summary>
         /// Gets the host key.
         /// </summary>
-        public HostAlgorithm HostKey { get; private set; }
+        public HostAlgorithm HostKey
```

Comment:
What was wrong with the automatic property?

Reply:
I guess nothing. At least cant remember why I changed it and dont see any reason why it should be 🤷